arcade/gui/property.py

```diff
@@ -18,21 +18,41 @@ class _Obs(Generic[P]):
     def __init__(self, value: P):
         self.value = value
         # This will keep any added listener even if it is not referenced anymore and would be garbage collected
-        self.listeners: Set[Callable[[], Any]] = set()
+        self.listeners: Set[Callable[[Any, P], Any]] = set()
 
 
 class Property(Generic[P]):
     """
     An observable property which triggers observers when changed.
 
+... code-block:: python
```

Comment:
My bad, typo in my last comment. It should be a double `.`, not a triple. :sweat_smile: 
```suggestion
.. code-block:: python
```